### ISSUE TITLE
Some problem when running the 5.XSeg) train.bat

### ISSUE THREAD

**Issue by TLSteve** [iperov/DeepFaceLab#855]
THIS IS NOT TECH SUPPORT FOR NEWBIE FAKERS
POST ONLY ISSUES RELATED TO BUGS OR CODE
okay
## Expected behavior
I run the "5.XSeg) train.bat" and it failed. Although i tried to delete some samples or change the videos, it failed... :(
## Actual behavior
the wrong message is
Caused by op 'conv2d_transpose_2', defined at:
  File "threading.py", line 884, in _bootstrap
  File "threading.py", line 916, in _bootstrap_inner
  File "threading.py", line 864, in run
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\mainscripts\Trainer.py", line 57, in trainerThread
    debug=debug,
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\models\Model_XSeg\Model.py", line 17, in __init__
    super().__init__(*args, force_model_class_name='XSeg', **kwargs)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\models\ModelBase.py", line 188, in __init__
    self.on_initialize()
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\models\Model_XSeg\Model.py", line 93, in on_initialize
    gpu_pred_logits_t, gpu_pred_t = self.model.flow(gpu_input_t)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\facelib\XSegNet.py", line 79, in flow
    return self.model(x)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\core\leras\models\ModelBase.py", line 117, in __call__
    return self.forward(*args, **kwargs)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\core\leras\models\XSeg.py", line 121, in forward
    x = self.up2(x)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\core\leras\models\ModelBase.py", line 117, in __call__
    return self.forward(*args, **kwargs)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\core\leras\models\XSeg.py", line 27, in forward
    x = self.conv(x)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\core\leras\layers\LayerBase.py", line 14, in __call__
    return self.forward(*args, **kwargs)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\core\leras\layers\Conv2DTranspose.py", line 81, in forward
    x = tf.nn.conv2d_transpose(x, weight, output_shape, strides, padding=self.padding, data_format=nn.data_format)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\python-3.6.8\lib\site-packages\tensorflow\python\ops\nn_ops.py", line 1654, in conv2d_transpose
    name=name)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\python-3.6.8\lib\site-packages\tensorflow\python\ops\gen_nn_ops.py", line 1307, in conv2d_backprop_input
    name=name)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\python-3.6.8\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 788, in _apply_op_helper
    op_def=op_def)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\python-3.6.8\lib\site-packages\tensorflow\python\util\deprecation.py", line 507, in new_func
    return func(*args, **kwargs)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\python-3.6.8\lib\site-packages\tensorflow\python\framework\ops.py", line 3300, in create_op
    op_def=op_def)
  File "[My File Location]\DeepFaceLab_NVIDIA\_internal\python-3.6.8\lib\site-packages\tensorflow\python\framework\ops.py", line 1801, in __init__
    self._traceback = tf_stack.extract_stack()

ResourceExhaustedError (see above for traceback): OOM when allocating tensor with shape[4,128,65,65] and type float on /job:localhost/replica:0/task:0/device:GPU:0 by allocator GPU_0_bfc
         [[node conv2d_transpose_2 (defined at [My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\core\leras\layers\Conv2DTranspose.py:81) ]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

         [[node Mean_33 (defined at [My File Location]\DeepFaceLab_NVIDIA\_internal\DeepFaceLab\models\Model_XSeg\Model.py:105) ]]

## Other relevant information
- **Operating system and version:Windows
- **Python version: 3.6.8

**Comment by z83rulek** [comment]
OOM means your memory is not enough

**Comment by ghost** [comment]
Issue solved / already answered (or it seems like user error), please close it.
